src/hex_tarball.erl

```diff
@@ -162,6 +230,20 @@ format_error({checksum_mismatch, ExpectedChecksum, ActualChecksum}) ->
         "Actual   (base16-encoded): ~s",
         [encode_base16(ExpectedChecksum), encode_base16(ActualChecksum)]).
 
+format_error(_Metadata, {Field, is_required}) -> <<Field/binary, <<" is a required metadata field">>/binary >>;
```

Comment:
Can we flip these around to `{is_required, Field}`?

Reply:
tagged tuples make sense 👍